yield_plugin/include/yield_plugin/yield_plugin.h

```diff
@@ -207,6 +208,14 @@ class YieldPlugin
    */
   void lookupECEFtoMapTransform();
 
+  /**
+   * \brief checks trajectory for minimum gap associated with it
+   * \param original_tp original trajectory plan
+   * \return minumum required
+   */
+  double check_traj_for_digital_min_gap(const cav_msgs::TrajectoryPlan& original_tp);
```

Comment:
I'd recommend adding a note to this comment that it returns `std::numeric_limits<double>::max()` if no such limit can be found, just so that anyone who might call this is aware later.